pkgs/development/compilers/dart/package-source-builders/media_kit_libs_linux/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv
+}:
+
+{version, src, ...}:
+
+stdenv.mkDerivation {
+  pname = "media_kit_libs_linux";
+  inherit version src;
+  inherit (src) passthru;
+  doBuild = false;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out"
+    cp -r '${src}'/* "$out"
+    chmod -R 777 $out/linux/CMakeLists.txt
+    cp ${./CMakeLists.txt} $out/linux/CMakeLists.txt
```

Comment:
Could this be made into a patch with [bracket comments](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-comment)? Or is the diff too big?

Reply:
Not sure. The src of that is not pivoted to the root of [1] so it would be harder to maintain a patch this way.

[1] https://github.com/media-kit/media-kit

Reply:
BTW this download part is conveniently enclosed with `# -----` things

Reply:
This way is better to maintain lol.

Reply:
I did with awk. Much simpler and awk is already in the stdenv so no extra deps.